book/src/installers/usage.md

```diff
@@ -0,0 +1,123 @@
+
+    You can use a custom environment variable to specify your install
+    location. If you do, that variable will be sourced during the install task. For more information, see the [documentation for this feature][install-path].
+
+- `$CARGO_HOME`
+
+    This environment variable is sourced if you use the `CARGO_HOME` install
+    location. For more information, see the [documentation for this feature][install-path].
+
+## `$PATH`
+
+> since 0.1.0, path modification options updated in 0.23.0
+
+`$PATH` is an environment variable that pre-exists on nearly all systems and
+lists locations to look for executables in. This is what allows you to call a
+program by its name instead of needing to call it by it's specific location on
+the file system.
+
+When you use axo installers, we do the heavy lifting of ensuring that your
+application is available "on PATH" to your end users. On Linux and macOS, we do this by editing shell dotfiles; on Windows, we do this by editing the `Environment.Path` registry key. However, there are
+circumstances where this is not desirable, and so we provide the ability to
+skip this setup step.
+
+- ### `$PATH`
+
+    This environment variable is both sourced and modified in the standard mode
+    of an installation. We source this variable to see if the installation
+    location is already on PATH and we will modify it if it is not.
+
+- ### `$PATH` modification
+
+    If you do not want your PATH to be modified you can use the `INSTALLER_NO_MODIFY_PATH` environment variable to configure your installation experience.
+
+- ### `$GITHUB_PATH`
+
+    If an installer detects the presence of this environment variable
+    (signalling that it is running in a GitHub Actions context) our installers
+    will modify this environment variable to ensure that all installed
+    applications are immediately available on PATH.
+
+## Receipt
+
+> since 0.9.0
+
+When you use axo to distribute your application, in addition to installers, you
+may also enable an updater - either integrated into your application using a
+library or as a standalone binary shipped alongside your application.
+
+The updater functionality relies on knowing how your application was originally
+installed and where. To keep track of this information, the installer writes a
+receipt that is read by the the updater.
+
+You can configure this receipt writing using the following options:
+
+- Shell: The `$HOME` environment variable is sourced to write the receipt to
+  `$HOME/.config/{app name}`.
+- PowerShell: The `$LOCALAPPDATA` environment variable is sourced to write the
+  receipt to `$LOCALAPPDATA/{app name}`.
+
+## Unmanaged mode
+
+> since 0.23.0
+
+This is intended for users installing in ephemeral install methods such as CI and disables several features that are unneeded in those environments. To use it, set the `$APPNAME_UNMANAGED_INSTALL` environment variable to the desired installation path. Enabling this mode does the following things:
```

Comment:
we should probably give this the treatment we gave above re: explaining how to replace `appname` with the app's name. could probably cut and paste even.

Reply:
Good call, done.